rero_ils/modules/locations/jsonschemas/form_locations/location-v0.0.1.json

```diff
@@ -14,7 +14,8 @@
     "type": "fieldset",
     "items": [
       {
-        "key": "name"
+        "key": "name",
+        "autofocus": "true"
```

Comment:
As JSON support boolean, I would replace the string `"true"` by a boolean `true`. You can fix it to the other files.

Reply:
done